README.md

```diff
@@ -149,9 +149,18 @@ Notes for developers
 
 To start contributing to Mosh, install the following dependencies:
 
+Debian, Windows Subsystem for Linux:
+
+```
+$ sudo apt install -y build-essential debhelper-compat protobuf-compiler \
```

Comment:
I don't believe we need debhelper-compat outside of Debian packaging.

Reply:
Done.